VSRAD.DebugServer/IPC/Commands.cs

```diff
@@ -27,8 +31,9 @@ public static void WriteCommand(this IPCWriter writer, ICommand command)
                 case Execute _: commandType = 0; break;
                 case FetchMetadata _: commandType = 1; break;
                 case FetchResultRange _: commandType = 2; break;
-                case Deploy _: commandType = 3; break;
-                case ListEnvironmentVariables _: commandType = 4; break;
+                case PutFileCommand _: commandType = 3; break;
+                case Deploy _: commandType = 4; break;
+                case ListEnvironmentVariables _: commandType = 5; break;
                 default: throw new ArgumentException($"Unable to serialize {command.GetType()}");
             }
             writer.Write(commandType);
```

Comment:
It’s also better to use enum value assignment (in combination with my previous suggestion) and then cast `commandType` variable to `byte` type